src/routes/(app)/admin[[hash=admin_hash]]/config/+page.server.ts

```diff
@@ -87,11 +91,13 @@ export const actions = {
 				isBillingAddressMandatory: z.boolean({ coerce: true }),
 				displayNewsletterCommercialProspection: z.boolean({ coerce: true }),
 				cartMaxSeparateItems: z.number({ coerce: true }).int().default(0),
-				disableLanguageSelector: z.boolean({ coerce: true })
+				disableLanguageSelector: z.boolean({ coerce: true }),
+				contactModes: z.string().array()
 			})
 			.parse({
 				...Object.fromEntries(formData),
-				paymentMethods: formData.getAll('paymentMethods')
+				paymentMethods: formData.getAll('paymentMethods'),
+				contactModes
```

Comment:
```suggestion
				contactModes: formData.getAll("contactModes")
```

Do it like for paymentMethods just above, to avoid calling `JSON.parse` / avoid doing validation outside of `zod`

Reply:
Doesn't work as excepted if i remove `JSON.parse`

Reply:
if you change the frontend it should.

In the frontend you can create an `<input name="contactModes">` for each contact mode.